Winch/Util/UtilHelpers.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using UnityEngine;
+using Winch.Core;
+using Winch.Serialization;
+
+namespace Winch.Util;
+
+public static class UtilHelpers
```

Comment:
This class is a bit choppy but structure of common logic can be refined in future iterations